Core/src/IfcGeometryConverter/SplineConverter.h

```diff
@@ -565,6 +364,8 @@ namespace OpenInfraPlatform
 
 					std::shared_ptr<PlacementConverterT<IfcEntityTypesT>> placementConverter;
 
+					SplineUtilities splineUtilities;
+

```

Comment:
I'd propose to make the functions `static` and omit the need for a member here.

Reply:
Done in commit 4f9851b.
Now, all functions of SplineUtilities are static because they don't store information in member variables of SplineUtilities. As well, these functions don't call other non-static functions, thus your suggestion is comprehensible - thanks.